src/pages/FAQ.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { ContentPage } from 'components/Layout/PageWrapper'
+import { react as FAQ_CONTENT } from 'assets/md/FAQ.md'
```

Comment:
Why capitalized? FaqContent would be fine imo

Reply:
renamed